docs/agent_container_manual_deploy.md

```diff
@@ -0,0 +1,76 @@
+## Instructions for starting an agent in a container on Linux
+These instructions assume that the agent container you wish to deploy already exists and is in a container image repository somewhere.
+
+### Prerequisites
```

Comment:
The document doesnt seem to format correctly in github when I view it in rich diff mode. I wonder if it will render correctly when the PR is merged.

Reply:
Fixed the angle brackets, looks right to me now